manifest.json

```diff
@@ -53,6 +54,30 @@
       "isRequired": false,
       "isBackendOnly": false,
       "order": 30
+    },
+    "default_comment_on_ticket_reply": {
+      "title": "Ticket reply as comment",
+      "description": "Enable option to add Deskpro replies as issue comments when a YouTrack issue is linked to a Deskpro ticket",
+      "type": "boolean",
+      "isRequired": false,
+      "isBackendOnly": false,
+      "order": 40
+    },
+    "default_comment_on_ticket_note": {
+      "title": "Ticket note as comment",
+      "description": "Enable option to add Deskpro notes as issue comments when a YouTrack issue is linked to a Deskpro ticket",
+      "type": "boolean",
+      "isRequired": false,
+      "isBackendOnly": false,
+      "order": 50
+    },
+    "dont_add_comment_when_linking_issue": {
```

Comment:
We can now have a `default` value here: https://support.deskpro.com/en-US/guides/developers/app-settings

Reply:
done